exist-core/src/main/java/org/exist/http/servlets/AbstractExistHttpServlet.java

```diff
@@ -246,9 +246,9 @@ protected Subject authenticate(HttpServletRequest request, HttpServletResponse r
 	        }
         }
 
-        // Secondly try basic authentication
+        // Secondly try basic authentication if there is no Authorization header or the Authorization header does not indicate Basic auth
         final String auth = request.getHeader("Authorization");
-        if (auth == null && getDefaultUser() != null) {
+        if ((auth == null || !auth.startsWith("Basic ")) && getDefaultUser() != null) {
```

Comment:
the auth comperation should be case-insensitive to be specification compliant by writing it somewhat like this:
`!auth.toLowerCase().startsWith("basic ")`

Reply:
Good spot @reinhapa! I have now added the case-insensitive match